features/starting_density_matrix.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Input a density matrix as a starting point for the calculation
```

Comment:
@horsfielda I think you'll also find this interesting. I had to add a new feature, i.e. add the ability to add a starting density matrix, and so I've added another "Feature". It has two "Scenarios", the first one demonstrates that you get stuck in a local minimum if you start from the single particle density matrix. The second scenario shows that you can escape the local minimum by using an input density matrix that is close to the solution. From the analytical work in my thesis I know that the energy = 0 at U/|t|  = 4 for the local minimum and that the energy = -1/2 at U/|t| = 4 for the global minimum. I also know what that solution looks like so it was easy to make the input density matrix.